yt_dlp/extractor/rutube.py

```diff
@@ -129,8 +159,9 @@ def suitable(cls, url):
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
-        info = self._download_and_extract_info(video_id)
-        info['formats'] = self._download_and_extract_formats(video_id)
+        secret = self._match_valid_url(url).group('secret')
```

Comment:
Isn't it better to pass on all query?
```suggestion
        query = parse_qs(url)
```
or atleast, parse the secret like this - I see no reason to parse it with regex
```suggestion
        query = {'p': parse_qs(url).get('p')}
```
(`from utils import parse_qs`)

Reply:
Thanks!! It was so stupid of me to actually parse it with a regex. No the code looks prettier and the changeset has become even smaller.